persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/ConversationDAOImpl.kt

```diff
@@ -116,13 +116,26 @@ class ConversationDAOImpl(
             .map { it.map(conversationMapper::toModel) }
     }
 
-    override suspend fun getConversationByQualifiedID(qualifiedID: QualifiedIDEntity): Flow<ConversationEntity?> {
+    override suspend fun getConversationByQualifiedIDFlow(qualifiedID: QualifiedIDEntity): Flow<ConversationEntity?> {
```

Comment:
As agreed on last collective, shouldn't we rename to
```suggestion
    override suspend fun observeConversationEntityByQualifiedId(qualifiedID: QualifiedIDEntity): Flow<ConversationEntity?> {
```